Cargo.toml

```
@@ -0,0 +1,41 @@
+[package]
+name = "kolibri-embedded-gui"
+version = "0.0.0-alpha.1"
+edition = "2024"
```

```suggestion
edition = "2021"
```

If you apply this I'd consider this ready to merge :)